src/netex-convertor/handler.ts

```diff
@@ -64,7 +64,10 @@ export const netexConvertorHandler = async (event: S3Event): Promise<void> => {
             const fileName = generateFileName(s3FileName);
 
             await uploadToS3(generatedNetex, fileName);
-            console.info(`NeTEx generation complete for type ${type}`);
+
+            if (matchingData.nocCode !== 'IWBusCo') {
+                console.info(`NeTEx generation complete for type ${type}`);
+            }
```

Comment:
Do we want the `console.info('NeTEx generation starting for type ${type}...');` [line 55] in this same wrapper also?